config/locales/active_admin.de.yml

```diff
@@ -294,6 +294,7 @@ de:
         flash:
           article_online: "Dieser Artikel ist nun online"
           article_offline: "Dieser Artikel ist nun offline"
+          article_not_edited: "Artikel konnte nicht gefunden und verarbeitet werden."
```

Comment:
Calling the path `/admin/articles/8/set_page_online_offline` with an wrong id (wrong slug before).
Don't know why somebody would do this. Maybe with multiple tabs and already deleted record in one of them.
Or calling the url directly with something bad?

Fact is, it happened somehow :)